creator-node/src/monitors/syncHistory.js

```diff
@@ -1,65 +1,52 @@
-const redisClient = require('../redis')
 const SyncHistoryAggregator = require('../snapbackSM/syncHistoryAggregator')
 
-const get30DayRollingSyncSuccessCount = async () => {
+/**
+ * Get the total number of syncs that met 'status' per wallet per day for the given number of days
+ * @param {String} status Either 'success' or 'fail'
+ * @param {Number} days number of historical days to get data for
+ * @returns {Number} number of syncs per wallet per day that met the status
+ */
+const _getRollingSyncCount = async (status, days) => {
+  if (status !== 'success' && status !== 'fail') throw new Error('Invalid status for _getRollingSyncCount: must be "success" or "fail"')
```

Comment:
prob better to export `SYNC_STATES` from the aggregator and do `if (SYNC_STATES.includes(status)` check here, but your call

Reply:
if you do this, can you also update the err msg with the modes too?

Reply:
so it can be dynamically updated

Reply:
done